source/GameObjects.cpp

```diff
@@ -0,0 +1,458 @@
+/* GameData.cpp
+Copyright (c) 2021 by Michael Zahniser
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+*/
+
+#include "GameObjects.h"
+
+#include "Audio.h"
+#include "BatchShader.h"
+#include "Color.h"
```

Comment:
Yes initially, I had forward declarations in the header. But that turned out to be a bad idea since anyone including the header would need to include allll of the other headers too. So, I decided to just `#include` them directly in this header.